src/style.css

```diff
@@ -0,0 +1,102 @@
+body {
+  background-color: rgb(236, 236, 236);
+  display: flex;
+  justify-content: center;
+  margin-top: 5em;
+  margin-bottom: 5em;
+}
+
+* {
+  font-family: sans-serif;
+  font-size: 15px;
+  font-weight: 200;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+#to-do-list {
```

Comment:
It is preferably better to use class attribute instead of ID selector for multiple elements. Check on that. :recycle: